data/json/mapgen/microlab/microlab_generic.json

```
@@ -2,7 +2,7 @@
   {
     "type": "mapgen",
     "method": "json",
-    "om_terrain": [ [ "microlab_generic" ] ],
+    "om_terrain": [ [ "microlab_generic" ], [ "microlab_ratkin" ] ],
```

You added a new OMT to this using the syntax that says you're going to define the two adjacent to each other.  But you added no new mapgen data for it, so you've defined `mapgen_ratkin` to be just a blank OMT.  I'm guessing what you wanted was for it to be identical to `microlab_generic`?  If so, you can do that with the following change:
```suggestion
    "om_terrain": [ "microlab_generic", "microlab_ratkin" ],
```

---

Thank you! I don't know how long it would have taken me to realize that.